packages/react-reconciler/src/__tests__/ReactUse-test.js

```diff
@@ -558,6 +558,7 @@ describe('ReactUse', () => {
     }
   });
 
+  // @gate !enableSiblingPrerendering
```

Comment:
Would it help if I wrote the tests for when this is on?

Reply:
I have some more later in the stack, but perhaps I'll move a few into this PR